controllers/gslb_controller.go

```diff
@@ -85,6 +86,9 @@ func (r *GslbReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 	if err != nil {
 		return result.RequeueError(fmt.Errorf("resolving spec (%s)", err))
 	}
+	log.Debug().
```

Comment:
@kuritka, I noticed that gopkg ToString() preserves JSON pretty-print formatting, do you think it's possible to add possibility to strip resulting serialized JSON out of `\n` and `\t`?
Now it looks like this:
```
2021-03-25T15:31:16+01:00 DBG controllers/gslb_controller.go:92 > Resolved strategy Strategy="{\n\t\"type\": \"failover\",\n\t\"primaryGeoTag\": \"eu\",\n\t\"dnsTtlSeconds\": 30,\n\t\"splitBrainThresholdSeconds\": 300\n}"
```

Reply:
@somaritane,  see: [gopkg issue](https://github.com/AbsaOSS/gopkg/issues/8)